ddp-workspace/projects/ddp-testboston/src/assets/i18n/ar.json

```diff
@@ -33,9 +33,9 @@
   "MailingList": {
     "Title": "نموذج القائمة البريدية",
     "Questions": {
-      "Adult": "1. هل تبلغ من العمر 18 عامًا أو أكثر؟",
-      "Patient": "2. هل أنت أحد مرضى مستشفى بريجهام والنساء؟",
-      "Email": "3. ما هو عنوان البريد الإلكتروني الأفضل لك؟"
+      "Adult": "١. هل تبلغ من العمر 18 عامًا أو أكثر؟",
+      "Patient": "٢. هل أنت أحد مرضى مستشفى بريجهام والنساء؟",
+      "Email": "٣. ما هو عنوان البريد الإلكتروني الأفضل لك؟"
```

Comment:
I think it should be fine to keep numeric digits as digits instead of Arabic symbols?

Reply:
I had a chat with Pegah, she said that it would be nicer to use Arabic symbols instead of numeric digits.